docs/metobs-example.md

```diff
@@ -0,0 +1,23 @@
+
+Direct usage of `MetObs`, without using the `SMHI` client is possible.
```

Comment:
Could be nice to add examples of usage of the `get_data` method, e.g. get station, parameter call chain etc.

Reply:
So, I implemented this in [63c53b9](https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi/pull/4/commits/63c53b9fb6f84bacbdd687ae67bee672d4871204). @docNord, please have a look and see if you like it.